src/mavis/summary/summary.py

```diff
@@ -224,8 +225,8 @@ def annotate_dgv(bpps, dgv_regions_by_reference_name, distance=0):
                 refname = dgv_region.reference_object.name
             except AttributeError:
                 pass
-            bpp.data['dgv'] = '{}({}:{}-{})'.format(
-                dgv_region.name, refname, dgv_region.start, dgv_region.end
+            bpp.data['dgv'].append(
+                '{}({}:{}-{})'.format(dgv_region.name, refname, dgv_region.start, dgv_region.end)
             )
 
```

Comment:
Should you add something to collapse this back to a string after the loop is complete? otherwise we might run into issues during output